### PR TITLE
[Testsuite] Re-enable passing XFAIL'd tests.

### PR DESCRIPTION
**Pull request by JDevlieghere**

### CODE REVIEW

**Comment by JDevlieghere** [comment]
@swift-ci please test and merge

**Comment by adrian-prantl** [comment]
I assume you checked that none of these are XFAILed on llvm.org?

**Comment by JDevlieghere** [comment]
@adrian-prantl Yep!

File: packages/Python/lldbsuite/test/lang/swift/expression/class_constrained_protocol/TestClassConstrainedProtocol.py [hunk 1]
```diff
@@ -23,7 +23,6 @@ def test_extension_weak_self (self):
         self.build()
         self.do_self_test("Break here for weak self")
 
-    @expectedFailureAll(bugnumber="rdar://31822722")
```
Comment (dcci) [inline packages/Python/lldbsuite/test/lang/swift/expression/class_constrained_protocol/TestClassConstrainedProtocol.py:26]: This needs to be marked as swift-test (not your fault, but while you're around, please fix it).

**Comment by JDevlieghere** [comment]
@swift-ci please test and merge

**Pull request merged**